department-of-veterans-affairs/caseflow
# APPEALS-37989: Create Seed Data for Legacy Cases with MST/PACT statuses

Resolves [APPEALS-37989](https://jira.devops.va.gov/browse/APPEALS-37989)
Resolves bug [APPEALS-39109](https://jira.devops.va.gov/browse/APPEALS-39109)

# Description
Added Legacy Appeals with Cases and MST/PACT statuses to seed data file so judges/attorneys in Demo can edit special issues statuses. 

## Acceptance Criteria
- [ ] Cases are created for these Users:
- [ ] BVASRITCHIE
- [ ] BVASCASPER1
- [ ] BVAEBECKER
- [ ] BVAKKEELING  

## Testing Plan
1. On local or demo run `make reset` or `bundle exec rails runner scripts/run_seed_data_file.rb db/seeds/mst_pact_legacy_case_appeals.rb` to run the seed script.
2. Sign in as BVASRITCHIE and navigate to Queue.
3. There should be Legacy Appeals in the Queue with different MST/PACT statuses (MST, PACT, and MST/PACT).
4. Repeat steps 2-3 with each other user in the AC list.
